exec/RenderHtml.hs

```diff
@@ -88,12 +87,9 @@ instance (ToHtml p) => ToHtml' (ToHtmlDefault p) where
     toHtml' (ToHtmlDefault p) = toHtml p
 
 instance (FormPageView p) => ToHtml' (ToHtmlForm p) where
-    toHtml' (ToHtmlForm p) = unwrap2 $ do
-        v <- unwrap1 $ getForm "" (makeForm p)
+    toHtml' (ToHtmlForm p) = toHtml $ do
+        v <- return . runIdentity $ getForm "" (makeForm p)
```

Comment:
you mean with a `runIdentity`, but without `return`?  you're right.  will you change it or should I?